RNTester/RNTester/AppDelegate.m

```diff
@@ -18,8 +18,17 @@
 #import <React/RCTPushNotificationManager.h>
 #endif
 
-@interface AppDelegate() <RCTBridgeDelegate>
+#import <React/RCTSurfacePresenter.h>
+#import <React/RCTFabricSurfaceHostingProxyRootView.h>
 
+@interface AppDelegate() <RCTBridgeDelegate>{
+  RCTSurfacePresenter *_surfacePresenter;
+}
+@end
+
+// FIXME: this is a hack, remove when surfaces start correctly
```

Comment:
Could you open an issue then link that issue here? We'll look into it.

Reply:
Yes, will do that shortly.

Reply:
https://github.com/facebook/react-native/issues/23910

Reply:
let's put this link in the FIXME comment: `https://github.com/facebook/react-native/issues/23910`